packages/runtime/container-runtime/package.json

```diff
@@ -152,12 +152,12 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
-			"dist/test/**/*.*js"
+			"lib/test/**/*.*js"
```

Comment:
It looks like many packages have this same config (using dist) - do they all need to change too?

Reply:
I think it depends on if they are testing ESM or CJS or both.
I wasn't trying to fix all of the things. Just getting what I saw trying to find good reference when adding tests to core-interfaces.

There are 9 more files with `-r node_modules/@fluid-internal/mocha-test-setup`.
And more that have "dist/test/..." than "lib/test/...".
@DLehenbauer, do you know if there was a debt item to go and tidy?